server/controllers/calls.js

```diff
@@ -140,9 +168,7 @@ export function recordAttempt(req, res) {
             const { contact_id, campaign_id, status } = call.attributes;
             if (validateStatusForUpdate(newStatus, status)) {
               if (newStatus === 'HUNG_UP') {
-                return callsService.updateCallStatus({ id: call_id, status: newStatus })
-                  .then(updatedCall => res.status(200).json({ message: `call status updated to ${newStatus}`, call: updatedCall }))
-                  .catch(err => console.log('error updating status in calls controller: ', err));
+                return handleHangUpFlow(res, user_id, call_id, newStatus);
```

Comment:
Thanks for not adding to this bloated method!

Reply:
💃